addons/website_slides_survey/controllers/main.py

```diff
@@ -8,6 +8,19 @@
 class WebsiteSlidesSurvey(http.Controller):
     @http.route(['/slides_survey/certification/search_read'], type='json', auth='user', methods=['POST'], website=True)
     def slides_certification_search_read(self, fields):
+        can_create = request.env['survey.survey'].check_access_rights('create', raise_exception=False)
         return {
             'read_results': request.env['survey.survey'].search_read([('certificate', '=', True)], fields),
+            'can_create': can_create,
         }
+
+    @http.route(['/slides_survey/certification/get_group'], type='json', auth='user', methods=['POST'], website=True)
+    def slides_certification_can_write(self):
+        group = False
+        if request.env.user.has_group('survey.group_survey_user'):
+            group = 'user'
+        if request.env.user.has_group('survey.group_survey_manager'):
+            group = 'manager'
+        return {
+            'group': group,
+        }
```

Comment:
All this should be done either in controller before loading template, not getting group, but using acls.

Reply:
I think I can indeed pass the value of the groupe of the user in the template but what are acls? :p